src/PrestaShopBundle/Resources/views/Admin/Sell/CustomerService/MerchandiseReturn/Blocks/form.html.twig

```diff
@@ -0,0 +1,90 @@
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{{ form_start(merchandiseReturnForm, {action: path('admin_merchandise_returns_update', {'merchandiseReturnId': editableMerchandiseReturn.merchandiseReturnId.value})}) }}
+  <div class="card">
+  <h3 class="card-header">
+    <i class="material-icons">person</i>
+    {{ 'Merchandise returns'|trans({}, 'Admin.Orderscustomers.Feature') }}
+  </h3>
+  <div class="card-block row">
+    <div class="card-text">
+      {{ form_errors(merchandiseReturnForm) }}
+      <div class="form-group row">
+        <label for="merchandise_return_merchandise_return_order_state" class="form-control-label">
+          {{ 'Customer'|trans({}, 'Admin.Global') }}
+        </label>
+        <div class="col-sm form-control-value">
+          <p>{{ editableMerchandiseReturn.customerFullName }}</p>
+          <a href="{{ path('admin_customers_view', {'customerId': editableMerchandiseReturn.customerId.value }) }}" class="btn btn-outline-secondary">
+            {{ 'View details on the customer page.'|trans({}, 'Admin.Orderscustomers.Help') }}
+          </a>
+        </div>
+      </div>
+
+      <div class="form-group row">
+        <label for="merchandise_return_merchandise_return_order_state" class="form-control-label">
+          {{ 'Order'|trans({}, 'Admin.Global') }}
+        </label>
+        <div class="col-sm form-control-value">
+          <p>#{{ editableMerchandiseReturn.orderId.value }} from {{ editableMerchandiseReturn.orderDate.format('d/m/y') }}</p>
+          <a href="{{ path('admin_orders_view', {'orderId': editableMerchandiseReturn.orderId.value }) }}" class="btn btn-outline-secondary">
+            {{ 'View details on the order page.'|trans({}, 'Admin.Orderscustomers.Help') }}
+          </a>
+        </div>
+      </div>
+
+      <div class="form-group row">
+        <label for="merchandise_return_merchandise_return_order_state" class="form-control-label">
+          {{ 'Customer explanation'|trans({}, 'Admin.Orderscustomers.Feature') }}
+        </label>
+        <div class="col-sm form-control-value">
+          <p>{{ editableMerchandiseReturn.question }}</p>
+        </div>
+      </div>
+
+      {% block merchandise_return_form_rest %}
+        {{ form_widget(merchandiseReturnForm) }}
+      {% endblock %}
+    </div>
+  </div>
+  <div class="card-footer">
+    <a href="{{ path('admin_merchandise_returns_index') }}" class="btn btn-outline-secondary">
+      {{ 'Cancel'|trans({}, 'Admin.Actions') }}
+    </a>
+    <button class="btn btn-primary float-right">
+      {{ 'Save'|trans({}, 'Admin.Actions') }}
+    </button>
+  </div>
+</div>
+{{ form_end(merchandiseReturnForm) }}
+
+{% block merchandise_returns_products %}
+  <div class="row">
+    <div class="col-12">
+      {% include '@PrestaShop/Admin/Common/Grid/grid_panel.html.twig' with {'grid': merchandiseReturnsProductsGrid} %}
+    </div>
+  </div>
+{% endblock %}
```

Comment:
```suggestion
{{ form_start(merchandiseReturnForm, {action: path('admin_merchandise_returns_update', {'merchandiseReturnId': editableMerchandiseReturn.merchandiseReturnId.value})}) }}
  <div class="card">
  <h3 class="card-header">
    <i class="material-icons">person</i>
    {{ 'Merchandise returns'|trans({}, 'Admin.Orderscustomers.Feature') }}
  </h3>
  
  <div class="card-block row">
    <div class="card-text">
      {{ form_errors(merchandiseReturnForm) }}
      
      <div class="form-group row">
        <label for="merchandise_return_merchandise_return_order_state" class="form-control-label">
          {{ 'Customer'|trans({}, 'Admin.Global') }}
        </label>
        
        <div class="col-sm form-control-value">
          <p>{{ editableMerchandiseReturn.customerFullName }}</p>
          
          <a href="{{ path('admin_customers_view', {'customerId': editableMerchandiseReturn.customerId.value }) }}" class="btn btn-outline-secondary">
            {{ 'View details on the customer page.'|trans({}, 'Admin.Orderscustomers.Help') }}
          </a>
        </div>
      </div>

      <div class="form-group row">
        <label for="merchandise_return_merchandise_return_order_state" class="form-control-label">
          {{ 'Order'|trans({}, 'Admin.Global') }}
        </label>
        
        <div class="col-sm form-control-value">
          <p>#{{ editableMerchandiseReturn.orderId.value }} from {{ editableMerchandiseReturn.orderDate.format('d/m/y') }}</p>
          
          <a href="{{ path('admin_orders_view', {'orderId': editableMerchandiseReturn.orderId.value }) }}" class="btn btn-outline-secondary">
            {{ 'View details on the order page.'|trans({}, 'Admin.Orderscustomers.Help') }}
          </a>
        </div>
      </div>

      <div class="form-group row">
        <label for="merchandise_return_merchandise_return_order_state" class="form-control-label">
          {{ 'Customer explanation'|trans({}, 'Admin.Orderscustomers.Feature') }}
        </label>
        
        <div class="col-sm form-control-value">
          <p>{{ editableMerchandiseReturn.question }}</p>
        </div>
      </div>

      {% block merchandise_return_form_rest %}
        {{ form_widget(merchandiseReturnForm) }}
      {% endblock %}
    </div>
  </div>
  
  <div class="card-footer">
    <a href="{{ path('admin_merchandise_returns_index') }}" class="btn btn-outline-secondary">
      {{ 'Cancel'|trans({}, 'Admin.Actions') }}
    </a>
    <button class="btn btn-primary float-right">
      {{ 'Save'|trans({}, 'Admin.Actions') }}
    </button>
  </div>
</div>
{{ form_end(merchandiseReturnForm) }}

{% block merchandise_returns_products %}
  <div class="row">
    <div class="col-12">
      {% include '@PrestaShop/Admin/Common/Grid/grid_panel.html.twig' with {'grid': merchandiseReturnsProductsGrid} %}
    </div>
  </div>
{% endblock %}
```

Just in order to improve readability if you don't mind

Reply:
Sure, this makes sense. Thanks!